qa/workload-manifests/dnsutils.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dnsutils
+  namespace: default
+spec:
+  securityContext:
+     runAsUser: 1000
+     runAsGroup: 3000
+     fsGroup: 2000
+  containers:
+  - name: dnsutils
+    image: registry.k8s.io/e2e-test-images/jessie-dnsutils:1.3
+    command:
+      - sleep
+      - "3600"
+    imagePullPolicy: IfNotPresent
+    securityContext:
+       allowPrivilegeEscalation: false
+  restartPolicy: Always
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dnsutils
+  namespace: kube-system
+spec:
+  securityContext:
+     runAsUser: 1000
+     runAsGroup: 3000
+     fsGroup: 2000
+  containers:
+  - name: dnsutils
+    image: registry.k8s.io/e2e-test-images/jessie-dnsutils:1.3
+    command:
+      - sleep
+      - "3600"
+    imagePullPolicy: IfNotPresent
+    securityContext:
+       allowPrivilegeEscalation: false
+  restartPolicy: Always
+
+  ## https://kubernetes.io/docs/tasks/administer-cluster/dns-debugging-resolution/
+  ## kubectl exec -ti dnsutils -- cat /etc/resolv.conf
+  ## kubectl exec -i -t dnsutils -- nslookup kubernetes.default
+
+#security context example setting
```

Comment:
Do these commented lines need to be removed?

Reply:
They're included links and example commands to ideally prevent someone from needing to leave their terminal in order to find additional information for how to use the tool.